corehq/motech/dhis2/tasks.py

```diff
@@ -88,20 +97,26 @@ def send_dataset(
                 'success': False,
                 'error': _('There was an error retrieving some UCR data. '
                            'Try contacting support to help resolve this issue.'),
-                'text': None
+                'text': None,
+                'log_url': response_log_url
             }
+
         except Exception as err:
             requests.notify_error(message=str(err),
                                   details=traceback.format_exc())
+            text = pformat_json(response.text if response else None)
+
             return {
                 'success': False,
                 'error': str(err),
                 'status_code': response.status_code if response else None,
-                'text': response.text if response else None,
+                'text': text,
+                'log_url': response_log_url
             }
         else:
             return {
                 'success': True,
                 'status_code': response.status_code,
-                'text': response.text,
+                'text': pformat_json(response.text),
+                'log_url': response_log_url
```

Comment:
Nit: I'm surprised this wasn't picked up by stickler, but we try and add trailing commas to keep the diff clean when adding new items to things like this (same for a few of the lines above too)

Reply:
Oh, that makes sense! I wondered why I kept seeing this pattern all over the codebase.